Sources/PointFree/VideoJs.swift

```diff
@@ -0,0 +1,40 @@
+import Css
+import Foundation
+import Html
+import HtmlCssSupport
+
+var videoJsHead: [ChildOf<Element.Head>] {
+  let videoJsAssets: [ChildOf<Element.Head>] = [
+    link([
+      href("https://cdnjs.cloudflare.com/ajax/libs/video.js/7.2.4/alt/video-js-cdn.min.css"),
+      rel(.stylesheet)
+      ]),
+    .init(script([
+      src("https://cdnjs.cloudflare.com/ajax/libs/video.js/7.2.4/video.min.js"),
+      `defer`(true)
```

Comment:
I think by marking this as `defer` I can keep this script in the head instead of injecting it at the bottom of the `body`. does that sound right?